tests/publiccloud/mr_test_setup_env.pm

```diff
@@ -0,0 +1,62 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Set up public cloud instance testing environment for 'mr_test'
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Tags: TEAM-6709
+
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use publiccloud::ssh_interactive qw(select_host_console prepare_ssh_tunnel);
+use testapi;
+use Mojo::File 'path';
+#use sles4sap_publiccloud;
+use publiccloud::utils;
+#use qesapdeployment;
```

Comment:
Any reason to leave these two lines commented?

Reply:
Thanks, deleted.